roles/init_dbserver/tasks/verify_tde.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Check if TDE is supported for EPAS version
+  ansible.builtin.fail:
+    msg: "TDE is not supported for Database Engine Version = {{ pg_version }}.
+          Supported versions are {{ supported_pg_version }}"
+  when: >
+    pg_version|int < 15 and pg_type == 'EPAS'
+
+- name: Verify edb_key_wrap_cmd is defined or not
+  ansible.builtin.fail:
+    msg: "edb_key_wrap_cmd and edb_key_unwrap_cmd are must for initializing a cluster"
+  when: >
+    edb_enable_tde and (edb_key_wrap_cmd|length < 1 or edb_key_unwrap_cmd|length < 1)
```

Comment:
I think that `edb_enable_tde` should be a condition to enter the `verify_tde.yml` tasks, so this should be redundant so it should read 
```
  when: >
    edb_key_wrap_cmd|length < 1 or edb_key_unwrap_cmd|length < 1
```